presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,156 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add support for variable precision `TIME WITH TIME ZONE` type. ({issue}`4905`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds`. ({issue}`4344`)
+* Add support for {func}`extract` for `timestamp(p) with time zone` with values of `p` other than 3. ({issue}`4867`)
+* Add support for correlated subqueries in recursive queries. ({issue}`4877`)
+* Remove support for political time zones in `TIME WITH TIME ZONE` type. ({issue}`191`)
+* Report dynamic filters stats. ({issue}`4440`)
+* Improve query scalability when new nodes are added to cluster. ({issue}`4294`)
+* Improve latency when dynamic filtering is in use. ({issue}`4924`)
+* Remove deprecated `reorder_joins` session property. ({issue}`5027`)
+* Fix serialization of `NULL` values in `ROW`, `MAP` and `ARRAY` types for old Presto clients. ({issue}`4778`)
+* Fix failure when aggregation query contains duplicate expressions. ({issue}`4872`)
+* Fix compiler failure when querying timestamps with a precision greater than 6. ({issue}`4824`)
+* Change timestamp operations to match the SQL specification. The value of a `TIMESTAMP` type is not affected by the session
+  time zone. ({issue}`4799`)
+* Fix parsing failure of timestamps due to daylight saving changes. ({issue}`4799`)
+* Remove the `deprecated.legacy-timestamp` configuration property and the `legacy_timestamp` session property. ({issue}`4799`)
+* Preserve precision when applying `AT TIME ZONE` to values of type `TIMESTAMP`. ({issue}`4866`)
+* Fix failure when calling `EXTRACT` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `TIMESTAMP WITH TIME ZONE` type. ({issue}`4867`)
+* Fix query deadlock for connectors that wait for dynamic filters. ({issue}`4946`)
+* Add `optimizer_rule_stats` system table. ({issue}`4659`)
+* Fix failure when `TIME` or `TIMESTAMP` subtraction returns a negative value. ({issue}`4847`)
+* Fix failure when duplicate expressions appear in `DISTINCT` clause. ({issue}`4787`)
+* Fix failure for join queries during spilling or when available memory is low. ({issue}`4994`)
+* Fix issue where ``query_max_scan_physical_bytes`` session property was ignored if
+  ``query.max-scan-physical-bytes`` config property was not defined. ({issue}`5009`)
+
+## Security Changes
+
+* Add Salesforce password authentication. ({issue}`4372`)
+* Add support for interpolating secrets into access-control.properties. ({issue}`4854`)
```

Comment:
I am not familiar with the word `interpolating` in this context. Is it used typically for that?

Reply:
Yes, it comes from the "string interpolation" concept in many programming languages: https://en.wikipedia.org/wiki/String_interpolation